Sources/WebAuthn/Ceremonies/Shared/AuthenticatorData.swift

```diff
@@ -107,3 +114,27 @@ extension AuthenticatorData {
         return (data, length)
     }
 }
+
+/// A helper type to determine how many bytes were condumed when decoding CBOR items.
```

Comment:
```suggestion
/// A helper type to determine how many bytes were consumed when decoding CBOR items.
```

Reply:
Whoops, fixed! I don't seem to have permission to run the CI as I don't have contributor status yet — is that something you can kick off?